build_scripts/windows/scripts/build.cmd

```diff
@@ -88,25 +90,47 @@ if not exist %WIX_DIR% (
 robocopy %PYTHON_DIR% %BUILDING_DIR% /s /NFL /NDL
 
 :: Build & install all the packages with bdist_wheel
-%BUILDING_DIR%\python %~dp0build-packages.py %TEMP_SCRATCH_FOLDER% %REPO_ROOT%
+%BUILDING_DIR%\python.exe -m pip install wheel
+echo Building CLI packages...
+set CLI_SRC=%REPO_ROOT%\src
+for %%a in (%CLI_SRC%\azure-cli %CLI_SRC%\azure-cli-core %CLI_SRC%\azure-cli-nspkg) do (
+   pushd %%a
+   %BUILDING_DIR%\python.exe setup.py bdist_wheel -d %TEMP_SCRATCH_FOLDER%
+   popd
+)
+pushd %CLI_SRC%\command_modules
+for /D %%a in (*) do (
+   pushd %CLI_SRC%\command_modules\%%a
+   %BUILDING_DIR%\python.exe setup.py bdist_wheel -d %TEMP_SCRATCH_FOLDER%
+   popd
+)
+popd
+echo Built CLI packages successfully.
+
 if %errorlevel% neq 0 goto ERROR
-:: Install them to the temp folder so to be packaged 
-%BUILDING_DIR%\python.exe -m pip install -f %TEMP_SCRATCH_FOLDER% --no-cache-dir azure-cli
+
```

Comment:
You can remove a couple of empty lines here

Reply:
Can do.
Will do in next CI so I don't have to wait for CI to pass again.